libs/@guardian/identity-auth/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # @guardian/identity-auth
 
+## 0.4.0
+
+### Minor Changes
```

Comment:
if peerDependencies change, [it should trigger a major release](https://github.com/guardian/recommendations/blob/main/npm-packages.md#changes-to-peerdependencies-ranges-are-breaking) rather than a minor one

Reply:
[Minor version changes before v1 are allowed to have breaking changes](https://semver.org/#spec-item-4)

Reply:
Thanks Max. We're treating `identity-auth` as pre-release hence why its minor. See comment below on whether its worth doing though.

Reply:
ah yes sorry, the others are all major versions 👍